csi/moac/test/volumes_test.js

```diff
@@ -1501,7 +1671,7 @@ module.exports = function () {
       expect(volume.state).to.equal('healthy');
       expect(Object.keys(volume.replicas)).to.have.length(3);
       // 2 nexus events
```

Comment:
Comment needs updating to "3 nexus events"

Reply:
thanks 👍